Sources/Code/UIViewController+Extension.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UIViewController+Extension.swift
+//  Imperio-iOS
+//
+//  Created by Anh Dung Pham on 15.08.17.
+//  Copyright © 2017 Flinesoft. All rights reserved.
+//
+
+import UIKit
+
+extension UIViewController {
+    class func currentViewController(_ viewController: UIViewController? = UIApplication.shared.keyWindow?.rootViewController) -> UIViewController? {
+        guard let viewController = viewController else { return nil }
+
+        if let viewController = viewController as? UINavigationController {
+            if let viewController = viewController.visibleViewController {
```

Comment:
The name of the `viewController` constant should be different than the outer `viewController` and should clarify that it's a `UINavigationController`.